### PR TITLE
feat: add switching workspace mode

### PR DESCRIPTION
**Pull request by ulbqb**
This PR adds switching workspace mode and then enables to generate a template without workspace. 

check command:
```
cargo generate --git https://github.com/Finschia/cw-workspace.git contracts -b feat/switch_ws_mode -d workspace=false
```

### CODE REVIEW

File: contracts/cw20/Cargo.toml [hunk 1]
```diff
@@ -13,12 +13,19 @@ backtraces = ["cosmwasm-std/backtraces"]
...
 [dependencies]
-cosmwasm-schema = { workspace = true }
+{% if workspace == "false" %}cosmwasm-schema = "=1.1.9"
+cosmwasm-std = "=1.1.9"
+cw-utils = "=1.0.1"
+cw2 = "=1.1.0"
+cw20 = "=1.1.0"
+cw-storage-plus = "=1.1.0"
+{% else %}cosmwasm-schema = { workspace = true }
+cosmwasm-std = { workspace = true }
```
Comment (zemyblue) [inline contracts/cw20/Cargo.toml]: How about using cargo-generate condition grammer and Rhai extensions?
https://cargo-generate.github.io/cargo-generate/templates/scripting.mini-example.html

Comment (ulbqb) [reply contracts/cw20/Cargo.toml]: I'll try it.

**Review by 170210 — APPROVED** [review]
LGTM

File: README.md [hunk 2]
```diff
@@ -26,8 +26,18 @@ cargo generate \
...
 ```
 
+## How To Use (without workspace)
+
+Generate a contract template without workspace.
+
+```
+cargo generate \
+    --git https://github.com/Finschia/cw-workspace.git contracts \
+    -d workspace=false
```
Comment (zemyblue) [inline README.md:36]: Can we use without `-d workspace=false`?
If we do not use -d option, can we use single mode?

Comment (ulbqb) [reply README.md:36]: If you use without `-d workspace=false`, you use workspace mode.
This project is for workspace. Single mode is optional.

Comment (170210) [reply README.md:36]: Could we use placeholder to instead of `-d`?  like after cargo-generate, there's an option (single|workspace).

Comment (ulbqb) [reply README.md:36]: I don't recommend it. It makes this complicated because users must understand the difference between single and workspace.

**Review by zemyblue — APPROVED** [review]

**Pull request merged**